src/utils/workpackageHelper.js

```diff
@@ -89,4 +90,101 @@ export const workpackageHelper = {
 			fileId: workPackage.fileId,
 		}
 	},
+	chunkMultipleSelectedFilesInformation(fileInformation) {
+		// this function chunks 20 files information in an array and returns array of it
```

Comment:
Can we also add a little bit of context on why exactly  `20` so that we are clear in future as well?
```suggestion
		// this function chunks 20 files of information in an array and returns an array of it, `20` because Openproject api only allows linking of 20 files per request
```